src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,430 @@
+| ChunkTransformationMatrixAxis | REQUIRED if `ChunkTransformationMatrix` is present  | [arrays][] of [strings][]              | Describe the axis of the ChunkTransformationMatrix (for examples: `["X", "Y"]` or `["Z", "Y", "X"]`)                                                                                                                                                                                                                                                                                                                                                                                   |
+
+An example of chunk transformations JSON metadata for `chunk-01` and `chunk-05` of Figure 2
+is shown below:
+
+![Figure 2](images/microscopy_chunks2.png "Example figure for chunks transformations")
+
+Figure 2: Example figure for chunks transformations.
+
+In this example, there is no scaling and `chunk-01` is at the origin.
+`chunk-05` is translated by 5 um along `X+` and by 3 um along `Y+`.
+
+**`*_chunk-01_<modality_suffix>.json`**:
+```JSON
+{
+        "PixelSize": [1, 1],
+        "PixelSizeUnits": "um",
+        "ChunkTransformationMatrix": [[1, 0, 0],
+                                      [0, 1, 0],
+                                      [0, 0, 1]],
+        "ChunkTransformationMatrixAxis": ["X", "Y"]
+}
+```
+
+**`*_chunk-05_<modality_suffix>.json`**:
+```JSON
+{
+        "PixelSize": [1, 1],
+        "PixelSizeUnits": "um",
+        "ChunkTransformationMatrix": [[1, 0, 5],
+                                      [0, 1, 3],
+                                      [0, 0, 1]],
+        "ChunkTransformationMatrixAxis": ["X", "Y"]
+}
+```
+
+#### Example of sidecar JSON file (`*_<modality_suffix>.json`)
+```JSON
+{
+        "Manufacturer": "Hamamatsu",
+        "ManufacturersModelName": "C9600-12",
+        "PixelSize": [0.23, 0.23],
+        "PixelSizeUnits": "um",
+        "Magnification": 40,
+        "BodyPart": "BRAIN",
+        "BodyPartDetails": "corpus callosum",
+        "SampleEnvironment": "exvivo",
+        "SampleFixation": "4% paraformaldehyde, 2% glutaraldehyde",
+        "SampleStaining": "LFB",
+        "SliceThickness": 5,
+        "TissueDeformationScaling": 97
+}
+```
+
+## Required Samples file (`samples.tsv`)
+
+For Microscopy data, the [Samples file](../03-modality-agnostic-files.md#samples-file) is REQUIRED.
+
+Additional optional columns in `samples.tsv` MAY be used to describe other samples' attributes.
+
+## Recommended participant data (`participants.tsv`)
+
+As for other data types in BIDS, the [Participants file](../03-modality-agnostic-files.md#participants-file)
+is RECOMMENDED and MUST contain the column `participant_id`.
```

Comment:
```suggestion
As for other data types in BIDS, the [Participants file](../03-modality-agnostic-files.md#participants-file)`participants.tsv`and its associated sidecar `participants.json` file
are RECOMMENDED and MUST contain the `participant_id` field.
```

Reply:
I updated the section to include your feedback. However, I don't think `participant_id` is mandatory in participants.json (from the example [here](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file)), so I kept it as a "column" of participants.tsv.

In addition, I added an example of particpants.json with the species, strain and strain_rrid definitions.
I would not add more details to these sections to avoid too much duplication with the general Samples file and Participants file sections of the spec.

Reply:
The `participants.tsv` is not recommended, it MUST be present and MUST have the `participant_id` column. The `participants.json` is not required but RECOMMENDED and, if present, may contain one or more fields describing the columns found in the TSV. So it is not required for the json to describe the participant_id field and your example is fine. But it should be more clear that the requirements on the presence and content of the tsv and json differ. We don't want people to get the impression that the tsv is optional as well.

Reply:
@robertoostenveld - the current specs say otherwise: https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file

Reply:
Hi @robertoostenveld, as @satra mentioned, the participants.tsv file is recommend in the main BIDS spec:
```
The purpose of this RECOMMENDED file is to describe properties of participants such as age, sex, handedness. If this file exists, it MUST contain the column participant_id, which MUST consist of sub-<label> values identifying one row for each participant, followed by a list of optional columns describing participants.
```
I don't think microscopy should go against the main spec. But we added this section to highlight the difference between samples.tsv and participants.tsv and also recommend the use of species, strain and strain_rrid.

> So it is not required for the json to describe the participant_id field and your example is fine. But it should be more clear that the requirements on the presence and content of the tsv and json differ. 

Similarly, I feel like the requirements on the content of tsv vs. json should be detailed in the main Participants file section and not in microscopy specifically. I would also prefer to avoid duplication as much as possible as it can create confusion for the user.

Reply:
🤨 that is new too me. I thought (and had always misread the spec, which states "file" whereas it actually refers to two files) that only the json was optional. 

I agree that documentation should where possible be at the highest level. I checked other modality specific sections, none of them mentions the `participants.tsv` files.

Reply:
Thanks @robertoostenveld for the feedback.

I followed yours and @guiomar suggestion to clarify the sample and participant sections. It ended up with more duplication  with the higher level which may not be advisable. I would be in favor of going back to a simpler wording which only highlighted the specificity of microscopy data.

------------

### Required Samples file
For Microscopy data, the [Samples file](https://bids-specification--881.org.readthedocs.build/en/881/03-modality-agnostic-files.html#samples-file) `samples.tsv` is REQUIRED and its associated sidecar `samples.json` file is RECOMMENDED.

Additional optional columns in `samples.tsv` MAY be used to describe other samples' attributes.

### Recommended participant data
For Microscopy data, we RECOMMEND to make use of the columns `species`, `strain` and `strain_rrid` in the [Participants file](https://bids-specification--881.org.readthedocs.build/en/881/03-modality-agnostic-files.html#participants-file) when applicable.

Additional optional columns in `participants.tsv` MAY be used to describe other subjects' attributes.

For example:
```
participant_id species strain strain_rrid
sub-01 mus musculus C57BL/6J RRID:IMSR_JAX:000664
sub-02 mus musculus C57BL/6J RRID:IMSR_JAX:000664
```
------------

As for reference, [Genetic Info](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/08-genetic-descriptor.html#subject-naming-and-participants-file) and [PET](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#recommended-patient-data) also have sections mentioning modality specific field in the participants file.

@robertoostenveld and @guiomar, let me know if this "lighter" wording would be acceptable for you.
Thank you for your help!

Reply:
yes, I am happy with this (and apologies for causing confusion).

Reply:
Thanks @robertoostenveld for the feedback.

@guiomar, please let me know if that works for you as well, thank you!

Reply:
Thanks @mariehbourget It looks good to me as well :)

The only suggestion that I would made (either here or in the general participants.tsv/json), is to include an example of a **participants.json** so people would have a template to use the definitions of `species`, `strain` and `strain_rrid` (as there are now for `age`,`sex`, `handedness`, etc)

Reply:
Hi @robertoostenveld and @guiomar. I made the changes to the samples/participants question with the "lighter" wording but kept the json example with `species`, `strain` and `strain_rrid`.
I think it's a good compromise between "including examples" and "avoiding duplication".
Thanks!